db/repos/rounds.js

```diff
@@ -88,10 +88,11 @@ class RoundsRepository {
 	 * Update the votes of for a particular account.
 	 *
 	 * @param {string} address - Address of the account
-	 * @param {int} amount - Votes to update
+	 * @param {BigNumber} amount - Votes to update
 	 * @todo Add @returns tag
 	 */
 	updateVotes(address, amount) {
+		amount = amount ? amount.toString() : '0';
```

Comment:
Why we need that? Is `amount` not required? If yes we should make it required.

Reply:
@4miners yes amount is required, however, we write weird tests to support undefined amount values here https://github.com/LiskHQ/lisk/blob/133036a05e8eb09397586af0f4d2aadd4e1ebd8a/test/unit/db/repos/rounds.js#L290
previously this test was working, because amount was integer, now its bignum and i convert that to `toString` before we save. so let me know weather to remove this invalid test or to keep the exisisting code.